### PR TITLE
UPNumber input

### PR DESCRIPTION
**Pull request by wgw0**
# Added UP number text field input and on page display functionality.

1. Put UP number in and press enter
2. If text input does not start with "UP" it tells you Error
3. If UP number is accepted it tells you student data is being loaded for that UP number.

Not my css, ref:
[CSS Reference](https://uiverse.io/Satwinder04/pink-bat-77)

### CODE REVIEW

**Review by NikBit101 — CHANGES_REQUESTED** [review]
Looks good. I would recommend forcing an upper case automatically, so the user won't need to do it manually.

Put this inside the input element in up-input field: style="text-transform: uppercase;"

Change line 13 inside the ui.js file to this: if (!inputField.value.toUpperCase().startsWith("UP")) {

**Comment by m0urao** [comment]
In my opinion, I think we should follow the design that the page has, for example, the input for the modules. I think we should follow the same style, either we change the UP input or we change all the others to keep everything the same. Again, it's just an opinion.

**Review by m0urao — COMMENTED** [review]
I like it, however, I think we should follow the design that the page has, for example, the input for the modules. I think we should follow the same style, either we change the UP input or we change all the others to keep everything the same. Again, it's just an opinion.

File: index.html [hunk 1]
```diff
@@ -14,6 +14,15 @@ <h1>Portsmouth Degree&nbsp;Calculator</h1>
 
 <main>
 
+  <section id="up-number-input">
+    <h2>Input your UP number to get your data:</h2>
+    <div class="input-container">
+      <input type="text" id="up-input" required="UP">
+      <label for="input" class="label">Enter UP number:</label>
```
Comment (m0urao) [inline index.html]: Also, I think there's an overkill of information here. Like you have a header with "Input your UP number to get your data:" and then you say again the same thing in the input "Enter UPnumber". I would say this instead:
```<label for="input" class="label">UPXXXXXX</label>```

Like you know when you have those email inputs and they give you ```example@gmail.com```, that's what I mean.

**Comment by wgw0** [comment]
### Changes following review

1. Forced uppercase with Nik's code

2. Removed instructional header + altered text-input text

**Review by NikBit101 — APPROVED** [review]

**Pull request merged**